plugins/woocommerce/client/legacy/js/frontend/country-select.js

```diff
@@ -158,7 +158,7 @@ jQuery( function( $ ) {
 				$newstate = $( '<input type="text" />' )
 					.prop( 'id', input_id )
 					.prop( 'name', input_name )
-					.prop( 'placeholder', placeholder )
+					.prop( 'placeholder', input_classes.includes( 'input-with-wc-float-label' ) ? '' : placeholder )
```

Comment:
If the input has a floating label, prevent the `placeholder` attribute from getting a value after updating the country select box in the shipping calculator.